app/components/homepage-components/ServicesCard.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import React, { useState } from "react";
+import Image from "next/image";
+import { useTheme } from "next-themes";
+
+interface Service {
+  imageSrc: string;
+  imageHoverLight: string;
+  imageHoverDark: string;
+  title: string;
+  description: string;
+}
+
+interface ServicesCardProps {
+  service: Service;
+}
+
+const ServicesCard: React.FC<ServicesCardProps> = ({ service }) => {
+  const { theme } = useTheme();
+  const [isMouseOver, setIsMouseOver] = useState<boolean>(false);
+  return (
+    <div
+      className={`h-[18rem] w-[19rem] rounded-xl  px-6 py-7 shadow-lg  xl:w-full ${
+        isMouseOver
+          ? "bg-primaryLight dark:bg-primaryDark"
+          : "bg-white800 dark:bg-black300"
+      }`}
+      onMouseOver={() => setIsMouseOver(true)}
+      onMouseLeave={() => setIsMouseOver(false)}
+    >
+      <div
+        className={`flex h-16 w-16 items-center justify-center rounded-xl ${
+          isMouseOver ? " bg-white900" : "bg-primaryLight dark:bg-primaryDark"
+        }`}
+      >
+        <Image
+          src={
+            isMouseOver
+              ? isMouseOver && theme === "light"
+                ? service.imageHoverLight
+                : service.imageHoverDark
+              : service.imageSrc
```

Comment:
Same critique here.

Reply:
All fixed. Is way more readable now. thank you